pkg/cmd/cli/cmd/config.go

```diff
@@ -39,6 +41,20 @@ func NewCmdConfig(parentName, name string) *cobra.Command {
 	cmd.Short = "Change configuration files for the client"
 	cmd.Long = configLong
 	cmd.Example = fmt.Sprintf(configExample, parentName, name)
-
+	adjustCmdExamples(cmd, parentName, name)
 	return cmd
 }
+
+func adjustCmdExamples(cmd *cobra.Command, parentName string, name string) {
+	for _, subCmd := range cmd.Commands() {
+		adjustCmdExamples(subCmd, parentName, cmd.Name())
+	}
+	cmd.Example = strings.Replace(cmd.Example, "$ kubectl", "$ "+parentName, -1)
```

Comment:
how many of these are there? would it be impossible to do an upstream PR that used `%[1]s` in examples?